<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GleisbelegungWindow</class>
 <widget class="QMainWindow" name="GleisbelegungWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="settings_page">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <attribute name="headerDefaultSectionSize">
           <number>200</number>
          </attribute>
          <column>
           <property name="text">
            <string>Gleis</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Konfliktgruppe</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Anschlüsse</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <item>
            <property name="text">
             <string>EA</string>
            </property>
            <property name="checkState">
             <enum>Checked</enum>
            </property>
            <item>
             <property name="text">
              <string>EA 1</string>
             </property>
             <property name="checkState">
              <enum>Checked</enum>
             </property>
             <property name="text">
              <string>EA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>EA 2</string>
             </property>
             <property name="checkState">
              <enum>Checked</enum>
             </property>
             <property name="text">
              <string>EA</string>
             </property>
            </item>
           </item>
          </item>
          <item>
           <property name="text">
            <string>Bahnhöfe</string>
           </property>
           <property name="checkState">
            <enum>PartiallyChecked</enum>
           </property>
           <item>
            <property name="text">
             <string>A</string>
            </property>
            <property name="checkState">
             <enum>PartiallyChecked</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flags">
             <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
            </property>
            <item>
             <property name="text">
              <string>A 1</string>
             </property>
             <property name="checkState">
              <enum>Checked</enum>
             </property>
             <property name="text">
              <string>A 1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>A 1 kurz</string>
             </property>
             <property name="checkState">
              <enum>Checked</enum>
             </property>
             <property name="text">
              <string>A 1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>A 2</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
             <property name="text">
              <string>A 2</string>
             </property>
            </item>
           </item>
           <item>
            <property name="text">
             <string>B</string>
            </property>
            <property name="checkState">
             <enum>PartiallyChecked</enum>
            </property>
            <item>
             <property name="text">
              <string>B 1</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
             <property name="text">
              <string>B1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B 2</string>
             </property>
             <property name="checkState">
              <enum>Unchecked</enum>
             </property>
             <property name="text">
              <string>B2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B 3a</string>
             </property>
             <property name="checkState">
              <enum>Checked</enum>
             </property>
             <property name="text">
              <string>B3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B 3b</string>
             </property>
             <property name="checkState">
              <enum>Checked</enum>
             </property>
             <property name="text">
              <string>B3</string>
             </property>
            </item>
           </item>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="display_page">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="displaySplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="grafikWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QLabel" name="zuginfoLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>Zuginfo: (keine Auswahl)</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSetup"/>
   <addaction name="actionAnzeige"/>
   <addaction name="separator"/>
   <addaction name="actionAnkunftAbwarten"/>
   <addaction name="actionAbfahrtAbwarten"/>
   <addaction name="actionPlusEins"/>
   <addaction name="actionMinusEins"/>
   <addaction name="actionFix"/>
   <addaction name="separator"/>
   <addaction name="actionLoeschen"/>
  </widget>
  <action name="actionSetup">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/equalizer.png</normaloff>:/equalizer.png</iconset>
   </property>
   <property name="text">
    <string>Setup</string>
   </property>
   <property name="toolTip">
    <string>Gleisauswahl (S)</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionAnzeige">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/chart.png</normaloff>:/chart.png</iconset>
   </property>
   <property name="text">
    <string>Grafik</string>
   </property>
   <property name="toolTip">
    <string>Grafik anzeigen (G)</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="actionPlusEins">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/hourglass--plus.png</normaloff>:/hourglass--plus.png</iconset>
   </property>
   <property name="text">
    <string>+1</string>
   </property>
   <property name="toolTip">
    <string>Feste Verspätung +1 Minute auf ausgewähltem Slot (+)</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionMinusEins">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/hourglass--minus.png</normaloff>:/hourglass--minus.png</iconset>
   </property>
   <property name="text">
    <string>-1</string>
   </property>
   <property name="toolTip">
    <string>Feste Verspätung -1 Minute auf ausgewähltem Slot (-)</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionFix">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/hourglass--pencil.png</normaloff>:/hourglass--pencil.png</iconset>
   </property>
   <property name="text">
    <string>Fix</string>
   </property>
   <property name="toolTip">
    <string>Feste Verspätung auf ausgewähltem Slot festlegen (V)</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="actionLoeschen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/eraser.png</normaloff>:/eraser.png</iconset>
   </property>
   <property name="text">
    <string>Löschen</string>
   </property>
   <property name="toolTip">
    <string>Korrekturen auf ausgewähltem Slot löschen (Del)</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAnkunftAbwarten">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/train.png</normaloff>:/train.png</iconset>
   </property>
   <property name="text">
    <string>Ankunft</string>
   </property>
   <property name="toolTip">
    <string>Kreuzung/Ankunft von zweitem gewählten Zug abwarten (K)</string>
   </property>
   <property name="shortcut">
    <string>K</string>
   </property>
  </action>
  <action name="actionAbfahrtAbwarten">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/train--arrow.png</normaloff>:/train--arrow.png</iconset>
   </property>
   <property name="text">
    <string>Abfahrt</string>
   </property>
   <property name="toolTip">
    <string>Überholung/Abfahrt von zweitem gewählten Zug abwarten (F)</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
